authentication/login.js

```diff
@@ -0,0 +1,37 @@
+
+const signUpForm = document.getElementById('sign-up'),
+trylogIn = document.getElementById('try-login'),
+trySignUp = document.getElementById('try-signup'),
+logInForm = document.getElementById('login');
+
+trySignUp.addEventListener('click', changeForm)
+trylogIn.addEventListener('click', changeForm)
+ 
+function changeForm(){
```

Comment:
You Can Use ES 6  Standard  It will Help you to Understand syntax

Reply:
Everything Else looks Good

Reply:
Made the changes